src/index.test.ts

```diff
@@ -33,5 +33,6 @@ describe('the default export', function () {
     expect(fake.digit()).toEqual(expect.any(Number));
     expect(fake.string()).toEqual(expect.any(String));
     expect(fake.uri()).toEqual(expect.any(String));
+    expect(fake.zip()).toEqual(expect.any(String));
```

Comment:
Probably not worth fixing now, but as you can see in the TODO at the top of this test, this awful "it works" mega-test for the entire module is tech debt that should not be added to. We should be moving to individual tests for individual methods (next to the file being tested) as we touch things.